internal/agent/hooks/logs.go

```diff
@@ -0,0 +1,47 @@
+package hook
+
+import (
+	"github.com/kairos-io/kairos-agent/v2/pkg/config"
+	"github.com/kairos-io/kairos-agent/v2/pkg/constants"
+	v1 "github.com/kairos-io/kairos-agent/v2/pkg/types/v1"
+	internalutils "github.com/kairos-io/kairos-agent/v2/pkg/utils"
+	fsutils "github.com/kairos-io/kairos-agent/v2/pkg/utils/fs"
+	"github.com/kairos-io/kairos-sdk/machine"
+	"path/filepath"
+	"syscall"
+)
+
+// CopyLogs copies all current logs to the persistent partition
+// useful during install to keep the livecd logs
+// best effort, no error handling
+type CopyLogs struct{}
+
+func (k CopyLogs) Run(c config.Config, _ v1.Spec) error {
```

Comment:
This tries to mount persistent and store the logs into int when the install is finished so we can keep the logs after install :D

Done on best effort, if it cant do something it just fails silently with no issues. we dont want this breaking an install. Its nice to have but if its impossible for some reason it will just be like "ok" and continue with its mundane life as a go binary